templates/faq.html

```diff
@@ -48,7 +48,9 @@ <h2 id="find-the-current-dns-servers">Find the current DNS servers</h2>
       <p>To determine the current DNS servers used by the system run <code>systemd-resolve --status</code> (or <code>resolvectl</code> in 18.04 and higher) and look for the &#39;DNS Servers:&#39; entry to see what DNS server is used.</p>
       <h2 id="deconfigure-an-interface">Deconfigure an interface</h2>
       <p>To deconfigure an interface, remove the configuration for the device from the netplan .yaml file and run <code>sudo netplan apply</code>.</p>
-      <p>If the interface is not configured in a .yaml file in <code>/etc/netplan</code>, it will not be configured at boot. To remove addresses manually, a user can run <code>ip address del &lt;address&gt; dev &lt;interface&gt;</code>.</p>
+      <p><strong>Note:</strong> <code>netplan apply</code> will not remove virtual devices such as bridges and bonds that have been created, even if they are no longer described in the Netplan configuration. That is due to the fact that Netplan operates statelessly and is not aware of the previously defined virtual devices.</p>
```

Comment:
```suggestion
      <p><strong>Note:</strong> <code>netplan apply</code> does not remove virtual devices, such as bridges and bonds, that have been created, even if they are no longer described in the Netplan configuration. That is because Netplan operates statelessly and is not aware of the previously defined virtual devices.</p>
```